pkg/collector/metric/vm_metric.go

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
This file is a good example that I am talking about introducing duplication, and we don't really need to do that.